client-common/Cargo.toml

```diff
@@ -19,6 +18,8 @@ chrono = { version = "0.4", features = ["serde"] }
 sled = { version = "0.24", optional = true }
 jsonrpc = { version = "0.11", optional = true }
 serde_json = { version = "1.0", optional = true }
+parity-codec = "3.5.1"
+parity-codec-derive = "3.3.0"
```

Comment:
Remove `parity-codec-derive` dependency and add `"derive"` feature in `parity-codec`.